datacapture/src/main/java/com/google/android/fhir/datacapture/views/factories/DropDownViewHolderFactory.kt

```diff
@@ -67,19 +68,26 @@ internal object DropDownViewHolderFactory :
           this.questionnaireViewItem.answerOption
             .map {
               DropDownAnswerOption(
+                it.value.identifierString(context),
                 it.value.displayString(context),
                 it.itemAnswerOptionImage(context)
               )
             }
             .toMutableList()
-        answerOptionList.add(0, DropDownAnswerOption(context.getString(R.string.hyphen), null))
+        answerOptionList.add(
+          0,
+          DropDownAnswerOption(
+            context.getString(R.string.hyphen),
+            context.getString(R.string.hyphen),
+            null
+          )
+        )
         val adapter =
           AnswerOptionDropDownArrayAdapter(context, R.layout.drop_down_list_item, answerOptionList)
-        val selectedAnswer =
-          questionnaireViewItem.answers.singleOrNull()?.value?.displayString(header.context)
+        val selectedAnswerIdentifier =
```

Comment:
alternative without using `idenitiferString`
```suggestion
        val selectedAnswerIdentifier =
          questionnaireViewItem.answers.singleOrNull()?.value?.let { it.id ?: it.displayString(context)}
```